server/proxy.ts

```diff
@@ -77,10 +80,10 @@ const proxyExternalHost = (host: any, accessToken: any, parseReqBody: any) =>
         (pathFromRequest ? pathFromRequest : "") +
         (queryString ? "?" + queryString : "");
 
-      if (host === Config.auth.isdialogmote.host) {
-        const newPathIsdialogmote = newPath.replace("isdialogmote/", "");
-        return newPathIsdialogmote;
+      if (removePathPrefix) {
```

Comment:
Ikke relatert til migreringen, men fikser https://nav-it.slack.com/archives/C012X71FN04/p1725275099559379 (Fjerner appnavnet fra første del av path når vi kaller api i `isdialogmote` og `modiacontextholder`)